service/src/main/resources/templates/ehr_observation_statement_effective_time_template.mustache

```diff
@@ -6,11 +6,14 @@
             {{{effectiveTime}}}
         </effectiveTime>
         <availabilityTime value="{{issued}}"/>
+        {{{value}}}
```

Comment:
If I'm reading the code correctly this template is only used by ObservationStatementMapper. ObservationStatementMapper maps unstructured FHIR Observation to HL7 ObservationStatement.

Maybe rename this file to `unstructured_observation_statement_template.mustache`?